### PR TITLE
docs: small fixes

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

**Comment by julio-b** [comment]
`g<C-A>` was removed in 83161200

```diff
diff --git a/runtime/doc/index.txt b/runtime/doc/index.txt
index c1a8aec40..969346c4c 100644
--- a/runtime/doc/index.txt
+++ b/runtime/doc/index.txt
@@ -699,7 +699,6 @@ tag		char	      note action in Normal mode	~
 
 tag		char	      note action in Normal mode	~
 ------------------------------------------------------------------------------ ~
-g_CTRL-A	g CTRL-A	   dump a memory profile
 |g_CTRL-G|	g CTRL-G	   show information about current cursor
 				   position
 |g_CTRL-H|	g CTRL-H	   start Select block mode

```

File: runtime/tutor/en/vim-01-beginner.tutor [hunk 1]
```diff
@@ -667,7 +667,7 @@ NOTE: If you were to exit Neovim and start it again with `nvim TEST`, the file
 
  4. Type
 
-        `:w TEST`{vim}
+        `w TEST`{vim}
```
Comment (zeertzjq) [inline runtime/tutor/en/vim-01-beginner.tutor:670]: I don't think this is correct

File: runtime/tutor/en/vim-01-beginner.tutor [hunk 2]
```diff
@@ -582,14 +582,14 @@ NOTE: You can also select the lines you want to substitute first using visual-mo
...
- 3. Typing `%`{normal} while the cursor is on a (,),[,],{, or } goes to its
+ 3. Typing `%`{normal} while the cursor is on a (, ), [, ], {, or } goes to its
     match.
 
  4. To substitute new for the first old in a line type
 ~~~ cmd
         :s/old/new
 ~~~
-    To substitute new for all 'old's on a line type
+    To substitute new for all olds on a line type
```
Comment (zeertzjq) [inline runtime/tutor/en/vim-01-beginner.tutor:592]: This is a bit strange

File: runtime/tutor/en/vim-01-beginner.tutor (hunk 1 above)
Comment (yndajas) [reply runtime/tutor/en/vim-01-beginner.tutor:670]: Hello! These changes are explained in more detail here: https://github.com/neovim/neovim/pull/27689

File: runtime/tutor/en/vim-01-beginner.tutor (hunk 2 above)
Comment (yndajas) [reply runtime/tutor/en/vim-01-beginner.tutor:592]: As per the other thread, these changes are explained in more detail here: https://github.com/neovim/neovim/pull/27689

File: runtime/tutor/en/vim-01-beginner.tutor (hunk 1 above)
Comment (yndajas) [reply runtime/tutor/en/vim-01-beginner.tutor:670]: Specifically for this one: https://github.com/neovim/neovim/commit/0ae1ec868fc1d843514fd799a9ade92c0fac7e8a

File: runtime/tutor/en/vim-01-beginner.tutor (hunk 2 above)
Comment (yndajas) [reply runtime/tutor/en/vim-01-beginner.tutor:592]: Specifically for this one: https://github.com/neovim/neovim/commit/c736321075e76708542d01edede38abfd4d6faa4

Comment (zeertzjq) [reply runtime/tutor/en/vim-01-beginner.tutor:592]: thanks

File: runtime/indent/scala.vim [hunk 3]
```diff
@@ -501,7 +501,7 @@ function! GetScalaIndent()
           let ind = indent(completeLine)
         endif
       else
-        " This is the only part that's different from from the '{', '}' one below
+        " This is the only part that's different from the '{', '}' one below
```
Comment (clason) [inline runtime/indent/scala.vim:504]:
```suggestion
```

We shouldn't mess with upstream files. OP should report it upstream instead.

**Pull request merged**